monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md

```diff
@@ -0,0 +1,56 @@
+# Portefueille Monero, démarrage rapide
+*22/04/2019*  
+Almutasim  
+_**Les portefeuilles impliquent l'une des décisions les plus importantes que doivent prendre les utilisateurs.**_
+
+Un portefeuille logiciel Monero est un programme permettant de stocker, envoyer et recevoir des Monero. Il vous donne le contrôle absolu sur vos Monero en toute confidentialité et sans intermédiaire. Personne ne peut connaître ou modifier l'ensemble des détails de vos transactions, à moins que vous n'ayez décidé de partager vos clefs de lecture à des fins d'audit, ce qui est l'essence même de l'expérience Monero.
+
+Pour faire l'expérience de ce pouvoir, vous devrez choisir un portefeuille. L'aspect le plus critique de tout portefeuille réside dans sa crédibilité. Seuls les portefeuilles recommandés par la communauté (et par le site [getmonero.org](https://getmonero.org/fr/)) sont mentionnés dans cet article. Ensuite, vous voudrez choisir un portefeuille compatible avec votre environnement préféré - iPhone, Android ou ordinateur. Enfin, vous devrez choisir un portefeuille qui répond à vos besoins de confidentialité. Cet article vous guidera sur ce point.
+
+Certains portefeuilles fonctionnent en utilisant leur propre copie de la chaîne de blocs de Monero. On appelle également cela faire fonctionner un nœud complet. Cela vous permet de contrôler la rapidité de vos transactions. Cependant, vous pouvez vous appuyer sur un nœud distant sans risquer perdre vos Monero, vous dépendez alors de la disponibilité du nœud distant (vous serez toujours en mesure de recevoir des Monero, mais vous pourriez ne pas pouvoir en envoyer). Un autre atout du fonctionnement de votre propre nœud complet réside dans la fierté de contribuer au réseau Monero. Les nœuds complets sincères rendent Monero plus fort. Apprenez-en plus ici : [monero.how/how-to-run-monero-node](https://www.monero.how/how-to-run-monero-node).
+
+S'appuyer sur des nœuds distant, au contraire, permet (en utilisant se qui est communément appelé un portefeuille léger) de s'affranchir des ressources de stockage nécessaires à la chaîne de bloc Monero. La chaîne de bloc (non élaguée) fait plus de 50Go et continue de grossir. De nombreux portefeuilles s'appuyant sur des nœuds complets distants utilisent également moins de puissance de calcul et de bande passante réseau. Si ces ressources sont limités sur votre appareil, un portefeuille léger pourrait être préférable. Cela sera presque toujours le cas pour un portefeuille fonctionnant sur Smartphone.
```

Comment:
Suggestion : 'La chaine de bloc non élaguée' ---> 'La blockchain complète'
Reasons: [1] blockchain is quite commonly used in French now. [2] 'non élaguée' is odd (although correct direct transcript of 'unpruned'). I think it is better to use the notion of full blockchain. Alternatively, may be replace 'élaguée' by 'tronquée'

Reply:
ok for "complète", not for blockchain (as i wrote in the first comment)